docs/blog/posts/2024/srlinux-asymmetric-routing.md

```diff
@@ -0,0 +1,1801 @@
+                        populate evpn {
+                        }
+                    }
+                    evpn {
+                        advertise dynamic {
+                        }
+                    }
+                }
+            }
+            anycast-gw {
+                anycast-gw-mac 00:00:5E:00:53:00
+            }
+        }
+    }
+```
+
+///
+
+/// tab | leaf4
+
+```{.srl .code-scroll-lg}
+--{ + running }--[  ]--
+A:leaf2# info interface irb0
+    interface irb0 {
+        admin-state enable
+        subinterface 10 {
+            admin-state enable
+            ipv4 {
+                admin-state enable
+                address 172.16.10.254/24 {
+                    anycast-gw true
+                }
+                arp {
+                    learn-unsolicited true
+                    proxy-arp true
+                    host-route {
+                        populate dynamic {
+                        }
+                        populate evpn {
+                        }
+                    }
+                    evpn {
+                        advertise dynamic {
+                        }
+                    }
+                }
+            }
+            anycast-gw {
+                anycast-gw-mac 00:00:5E:00:53:00
+            }
+        }
+        subinterface 20 {
+            admin-state enable
+            ipv4 {
+                admin-state enable
+                address 172.16.20.254/24 {
+                    anycast-gw true
+                }
+                arp {
+                    learn-unsolicited true
+                    host-route {
+                        populate dynamic {
+                        }
+                        populate evpn {
+                        }
+                    }
+                    evpn {
+                        advertise dynamic {
+                        }
+                    }
+                }
+            }
+            anycast-gw {
+                anycast-gw-mac 00:00:5E:00:53:00
+            }
+        }
+    }
+```
+
+///
+
+There is a lot going on here, so let's breakdown some of the configuration options:
+
+`anycast-gw [true|false]`
+
+:   When this is set to `true`, the IPv4 address is associated to the anycast gateway MAC address and this MAC address is used to respond to any ARP requests for that IPv4 address. This also allows the same IPv4 address to be configured on other nodes for the same broadcast domain, essentially suppressing duplicate IP detection.
+
+`anycast-gw anycast-gw-mac [mac-address]`
+
+:   The MAC address configured with this option is the anycast gateway MAC address and is associated to the IP address for that subinterface. If this is ommitted, the anycast gateway MAC address is auto-derived from the VRRP MAC address group range.
+
+`arp learn-unsolicited [true|false]`
+
+:   This enables the node to learn the IP-to-MAC binding from any ARP packet and not just ARP requests.
+
+`arp host-route populate [dynamic|static|evpn]`
```

Comment:
from jorge

This command is really not adding anything in asymmetric mode:
"arp host-route populate [dynamic|static|evpn]
This enables the node to insert a host route (/32 for IPv4 and /128 for IPv6) in the routing table from dynaimc, static or EVPN-learnt ARP entries."

the reason being that the routing is always done at ingress based on the destination subnet, and then based on the arp resolution. Having /32s serve no purpose. In symmetric mode it is important to avoid tromboning if multiple leaves are attached to the same subnet, but I don't see the use in asymmetric mode.

Reply:
agreed, I will remove this.